fawkes-buildenv/patches.2017.d/fawkes-robotino-gazebo-black-workaround.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/etc/scripts/gazsim-startup.bash b/etc/scripts/gazsim-startup.bash
+index e21ef0a0..78e2bed0 100755
+--- a/etc/scripts/gazsim-startup.bash
++++ b/etc/scripts/gazsim-startup.bash
+@@ -160,7 +160,7 @@ case $COMMAND in
+     gazebo )
+ 	# change Language (in german there is an error that gazebo can not use a number with comma)
+ 	export LC_ALL="C"
+-	gazebo $REPLAY $GAZEBO_WORLD $@
++  ( gzserver $REPLAY $GAZEBO_WORLD $@ & ); sleep 10s; gzclient
```

Comment:
You can't run gzclient on the cluster for one. For another, we do not use the gazsim-bash script. Why then add this patch?

Reply:
That's just the rebased version of [this patch](https://github.com/timn/docker-robotics/blob/master/fawkes-buildenv/patches.d/fawkes-robotino-gazebo-black-workaround.patch), which is included in the old image.

Reply:
I see, that was for the standalone version which we do not maintain anymore. Well, we can remove this with the 2016 version then later.